cpu/efm32/families/efm32gg/efm32-info.mk

```diff
@@ -0,0 +1,47 @@
+# This file is automatically generated, and should not be changed. There is
+# probably little reason to edit this file anyway, since it should already
+# contain all information for the EFM32GG family of CPUs.
```

Comment:
I'm trying to add support to another member of the efm32 family and now I wonder what tool was used to generate that kind of file.

Reply:
You might want to look here: https://github.com/basilfx/EFM2RIOT/tree/develop/dist

It should be reasonably up to date and contain all supported chips already.

Reply:
Uh that's nice.
@aabadie did something very similar for stm32.

I take we can't do that on the fly as Gecko SDK is required. But I don't like copying around folders either.

But we could just import that wholesale.

Reply:
I don't think it is necessary to do it on-the-fly. These families are quite stable (only new ones are added).

Reply:
So how about we import https://github.com/basilfx/EFM2RIOT/tree/develop/dist/cpu/efm32/families into RIOT as a whole?